backend/tracim_backend/lib/core/content.py

```diff
@@ -1450,6 +1462,36 @@ def copy(
         new_file_extension: str = None,
         do_save: bool = True,
         do_notify: bool = True,
+    ) -> Content:
+        """
+        Copy nearly all content, revision included. Children not included, see
```

Comment:
The docstring is wrong. You copy all contents, children included now.

Reply:
done.